src/components/proposals/proposal-list.vue

```diff
@@ -102,12 +102,13 @@ export default {
 
     voting (proposal) {
       if (proposal && proposal.votetally && proposal.votetally.length) {
-        const passCount = parseFloat(proposal.pass.count)
-        const failCount = parseFloat(proposal.fail.count)
+        // const passCount = parseFloat(proposal.pass.count)
+        // const failCount = parseFloat(proposal.fail.count)
         const abstain = parseFloat(proposal.votetally[0].abstain_votePower_a)
         const pass = parseFloat(proposal.votetally[0].pass_votePower_a)
         const fail = parseFloat(proposal.votetally[0].fail_votePower_a)
-        const unity = (passCount + failCount > 0) ? passCount / (passCount + failCount) : 0
+
+        const unity = (pass + fail > 0) ? pass / (pass + fail) : 0
         let supply = this.supply
```

Comment:
maybe for now OK but this duplicated code needs to be unified somewhere in a central location.

Reply:
Is it possible to quickly move this code to one location

We can't have the same calculation in different areas of the code, breaks the DRY rule (don't repeat yourself)

Reply:
Will be done later